src/tools/pub/cmd/pub-global.md

```diff
@@ -73,6 +73,15 @@ $ dart pub global activate --source git https://github.com/dart-lang/async_await
 $ dart pub global activate -sgit https://github.com/dart-lang/async_await.git
 ```
 
+Pub assumes that the package is in the root of the Git repository. 
+To specify a different location in the repo,
+use the `--git-path` flag to
+specify a path relative to the repository root:
```

Comment:
**question:** Do we use _flag_ or _option_? I would recommend option. (See [docopt standards](http://docopt.org/).

**suggestion:** 
```suggestion
Pub expects to find the package in the root of the Git repository.
To specify a different location, use the  `--git-path` option with
a path relative to the repo root.
```

Reply:
Using option makes sense and is consistent with the help menu. Thanks!